### PR TITLE
xml version updating

### PR DESCRIPTION
**Pull request by akshitdewan**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/Treeview.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/Treeview.xml]: Please execute `mvn xar:format` twice.

**Comment by tmortagne** [comment]
Thanks for your contribution !

**Comment by Enygma2002** [comment]
@tmortagne:There were some problems with this PR:
* 5 pages were already included in the previous PR https://github.com/xwiki/xwiki-platform/pull/659/files.
* there are changes on translation pages (other than the default language)
* there are changes on date fields

IMO, we should revert this PR as well and allow @akshitdewan to redo it.

**Comment by akshitdewan** [comment]
@Enygma2002 my apologies, I undid those 5 pages here because I forgot to make a new branch but they seem to have reappeared somehow. Nonetheless, it shouldn't affect anything because they are both merged right? Also, should I not be changing any translation pages? the bulk of the next PR is also translation pages. Finally, what is the point of excluding the update dates since it was, in fact, updated on this new date?

**Pull request merged**